controller/bookController.go

```diff
@@ -0,0 +1,45 @@
+package controller
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/joseantoniovz/academy-go-q12021/model"
+	"github.com/joseantoniovz/academy-go-q12021/util"
+)
+
+func loadData() []model.Book {
```

Comment:
The controller shouldn't be making calls to DB or external APIs, it should only receive and validate the request info and pass it over to the next layer where you can handle business logic and external dependencies calls.